src/fiskaltrust.AndroidLauncher.Common/fiskaltrust.AndroidLauncher.Common.csproj

```diff
@@ -107,7 +107,7 @@
     <AndroidAsset Include="Migrations\001_Init.sqlite3" />
     <AndroidAsset Include="Migrations\002_TransactionPersistence.sqlite3" />
     <AndroidAsset Include="Migrations\003_TseInfoJsonSignaturCreationUnitDE.sqlite3" />
-    <AndroidAsset Include="Migrations\004_FailedFinishTransactionRequest.sqlite3" />
+    <AndroidAsset Include="Migrations\004_FailedFinishTransactionRequest - Copy.sqlite3" />

```

Comment:
I know that it is only migration file, but should we remove -Copy from the name?

Reply:
Seems like Florian had the same thoughts 😄